CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing to the TODO-track-name
+
+We welcome contributions to the content and source code of the TODO-track-name.
+You may contribute by:
+
+* opening issues: https://github.com/security-summer-school/TODO-track-name/issues
+* taking part in discussions: https://github.com/security-summer-school/TODO-track-name/discussions
+* creating pull requests to update the content: https://github.com/security-summer-school/TODO-track-name//pulls
+
+When opening an issue, please clearly state the problem.
```

Comment:
I think that the content from here to the end of the file should be part of the future contributing guide in the [`meta` repository](https://github.com/security-summer-school/meta).